### ISSUE TITLE
dplug-build: moving binary fails

### ISSUE THREAD

**Issue by 0xEAB** [AuburnSounds/Dplug#275]
**OS:** Linux 4.17 (x86-64), Ubuntu 18.04
**Compiler:** LDC v1.9.0 / DMD v2.081.0
**dplug-build:** 13241b2e

## Output
[...]
```
distort ~master: building configuration "VST"...
    => Build OK, binary size = 7.7 mb, available in ./builds/Linux-64-bit-VST

error: Orphan format arguments: args[0..1]
```

## Result
`*.so` is not moved to `builds/`

**Comment by p0nce** [comment]
Mmm can't build dplug-build on Linux from this side.

**Comment by p0nce** [comment]
Fixed with 81f8f79c2830c97d93558108500ee3cb08598570

**Comment by 0xEAB** [comment]
thx :)
